src/lib/scss/fonts.scss

```diff
@@ -4,90 +4,97 @@
   font-style: normal;
   font-weight: 300;
   src: url('../fonts/roboto-v27-latin-300.eot'); /* IE9 Compat Modes */
-  src: local(''),
-       url('../fonts/roboto-v27-latin-300.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
-       url('../fonts/roboto-v27-latin-300.woff2') format('woff2'), /* Super Modern Browsers */
-       url('../fonts/roboto-v27-latin-300.woff') format('woff'), /* Modern Browsers */
-       url('../fonts/roboto-v27-latin-300.ttf') format('truetype'), /* Safari, Android, iOS */
-       url('../fonts/roboto-v27-latin-300.svg#Roboto') format('svg'); /* Legacy iOS */
+  src:
+    local(''),
+    url('../fonts/roboto-v27-latin-300.eot?#iefix') format('embedded-opentype'),
+    /* IE6-IE8 */ url('../fonts/roboto-v27-latin-300.woff2') format('woff2'),
+    /* Super Modern Browsers */ url('../fonts/roboto-v27-latin-300.woff') format('woff'),
+    /* Modern Browsers */ url('../fonts/roboto-v27-latin-300.ttf') format('truetype'),
+    /* Safari, Android, iOS */ url('../fonts/roboto-v27-latin-300.svg#Roboto') format('svg'); /* Legacy iOS */
```

Comment:
formatting is wrong in this file. Comments aren't related to the lines where they are placed

Reply:
@mavdryk reformat is updated.